src/events.ts

```diff
@@ -29,12 +30,12 @@ const messageBuilder = (context, event: EventResult): string => {
 
   if (received) {
     if (result) {
-      let resultJson = JSON.stringify(result);
+      let resultJson = JSON.stringify(parseArgs(context, result));
       let recordJson = JSON.stringify(record);
-      let isExpectedResult = isExpectedEventResult(event);
+      let isExpectedResult = isExpectedEventResult(context, event);
 
       if (isExpectedResult) {
-        resultMessage = `\n\n   ✔️  Expected Result: ${resultJson}`;
+        resultMessage = `\n\n   ✔️  Expected Result: ${resultJson} | Received Result: ${recordJson}`;
```

Comment:
useful when you wanna see full result to extend the expected

Reply:
Do you need this? Because I was think of a debug mode where all received events are printed.